### ISSUE TITLE
[RFE] Add the ability to download files from the web ui as well 

### ISSUE THREAD

**Issue by ghost** [fedora-infra/bodhi#874]
I have been working on this redesign of the Bodhi's web UI, the feedback I came across could be a good one to implement. The idea is to basically have 
1)  The koji download option in the web ui itself ( not redirecting to the another website ) 
 2) State out the command line link as well ( so that it can be cut copy pasted)

**Comment by trishnaguha** [comment]
This is a great idea indeed! I would like to work on it.
Should we have all the `RPMs` on the web ui of Bodhi?
Like the RPMs Koji displays: for example `src.rpm`, `noarch.rpm`,  `x86_64.rpm` and so on?

@lmacken Could you please share your thoughts on this? :)

**Comment by ghost** [comment]
The download files could be zip files, I am still understanding how to display the files in UI. So far we can display something like this : 
-source packages: packagename-src.tar.bz2 (100 MB)
- x86_64 packages: packagename-x86_64.tar.bz2 (48 MB) download all 
- ARM packages: packagename-ARM.tar.bz2 (37 MB)  download all 
  and so on. 
  However, this just an idea, once we have more concrete display options we can proceed with the development.

**Comment by ghost** [comment]
This is how I am thinking the widget would work and look like : 
![download1](https://cloud.githubusercontent.com/assets/14060749/17492659/c31c6d34-5d7a-11e6-96de-c40831ae7a94.png)
![download4](https://cloud.githubusercontent.com/assets/14060749/17492665/caac3c46-5d7a-11e6-925e-8bec871bb89f.png)

**Comment by trishnaguha** [comment]
@radhikamani88 Thanks for the update. 
@lmacken What are the things we should look into to have those packages directly as downloadable on Bodhi UI? 
Let me rephrase what we actually want:
- Bodhi should be able to check what type of packages have been built for the update and list them in the dropdown list.
- Selecting that package should return the package as downloadable along with the size.

So what are the things should we look into to have the type of packages listed on Bodhi? I am thinking to look into Pungi?
